Automattic/sensei
# Add recurring migration job

Resolves #5437 

## Proposed Changes
* Add WooCommerce Action Scheduler as a composer package and use it for the background jobs.
* Implement the migration job and the scheduler for it.

## Testing Instructions

1. Enable `tables_based_progress` feature.
5. Go to Sensei LMS -> Tools.
6. Run action for "Migrate comment-based student progress".
7. Refresh the page a few times until you see "Status: Completed." in the description of the tool.

Might be needed to do some cleanup if you tested migration before:
```
truncate table `wp_sensei_lms_progress`;
delete from `wp_options` where `option_name` = "sensei_migrated_progress_last_comment_id";
```

## Pre-Merge Checklist
